README.md

```diff
@@ -345,6 +346,33 @@ For more information see complete <a href="https://www.apify.com/docs/sdk/apify-
 
 <div id="include-readme-3">
 
+#### Puppeteer Live View
```

Comment:
I would link this paragraph from `liveView` parameter at `Apify.launchPuppeteer()`